tests/AdagucTests/TestWMS.py

```diff
@@ -55,15 +62,34 @@ def test_WMSGetCapabilities_testdatanc(self):
         AdagucTestTools().writetofile(self.testresultspath + filename,data.getvalue())
         self.assertEqual(status, 0)
         self.assertTrue(AdagucTestTools().compareGetCapabilitiesXML(self.testresultspath + filename, self.expectedoutputsspath + filename))
+        self.assertFalse(os.path.exists("checker_report.txt"))
 
     def test_WMSGetMap_testdatanc(self):
         AdagucTestTools().cleanTempDir()
         filename="test_WMSGetMap_testdatanc"
-        status,data,headers = AdagucTestTools().runADAGUCServer("source=testdata.nc&SERVICE=WMS&SERVICE=WMS&VERSION=1.3.0&REQUEST=GetMap&LAYERS=testdata&WIDTH=256&HEIGHT=256&CRS=EPSG%3A4326&BBOX=30,-30,75,30&STYLES=testdata%2Fnearest&FORMAT=image/png&TRANSPARENT=FALSE&", env = self.env)
+        status,data,headers = AdagucTestTools().runADAGUCServer("source=testdata.nc&SERVICE=WMS&SERVICE=WMS&VERSION=1.3.0&REQUEST=GetMap&LAYERS=testdata&WIDTH=256&HEIGHT=256&CRS=EPSG%3A4326&BBOX=30,-30,75,30&STYLES=testdata%2Fnearest&FORMAT=image/png&TRANSPARENT=FALSE&",
+                                                                env = self.env, args=["--report"])
         AdagucTestTools().writetofile(self.testresultspath + filename,data.getvalue())
         self.assertEqual(status, 0)
         self.assertEqual(data.getvalue(), AdagucTestTools().readfromfile(self.expectedoutputsspath + filename))
 
+        self.checkReport(reportFilename="checker_report.txt",
+                         expectedReportFilename="checker_report_WMSGetMap_testdatanc.txt")
+
+    def test_WMSGetMap_Report_env(self):
```

Comment:
Maybe we can also add a test to check if errors logged with either CDBError or CREPORT_ERROR_NODOC show up in the logs when reporter functionality is disabled.

Reply:
Done. Commit a484eb4677fe318b57decea33b506f547f31cebe